BusinessLogicLayer/Translation/TranslationHandler.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BusinessLogicLayer.Helpers;
+using BusinessLogicLayer.ViewModels;
+
+namespace BusinessLogicLayer.Translation
+{
+    public class TranslationHandler
+    {
+        private readonly ManageAccessToEntity _manageAccessToEntity;
+        private static int _languageNativeTranslationId = 0;
+        private static int _languageToLearnId = 0;
+        public TranslationHandler(ManageAccessToEntity manageAccessToEntity)
+        {
+            _manageAccessToEntity = manageAccessToEntity;
+        }
+
+        public List<ItemWithTranslation> GetLanguagesWithTranslations()
+        {
+            var languageWithTranslation = new List<ItemWithTranslation>();
+            var allLanguage = _manageAccessToEntity.Languages.GetAll();
+            var countryHandler = new CountryHandler(allLanguage);
+            var languageInCountry = countryHandler.TakeCountryByLanguage();
+            var allWords = _manageAccessToEntity.Words.GetAll();
+
+            foreach (var language in allLanguage)
+            {
+                languageWithTranslation.Add(new ItemWithTranslation()
```

Comment:
You can use auto-mapping here.
Please read about AutoMapper library and try to use it here.